src/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -10,8 +10,11 @@ export function VislibVisualizationsColumnChartProvider(Private) {
     mode: 'normal',
     showTooltip: true,
     color: undefined,
-    fillColor: undefined
+    fillColor: undefined,
   };
+
+  const minBarWidth = 1;
```

Comment:
one side-effect is now that when we make the visualization small enough, the error message becomes unreadable. 
![image](https://cloud.githubusercontent.com/assets/1833023/26593764/526bea14-4533-11e7-9007-47ce2b80b91c.png)